solidity/contracts/converter/LiquidityPoolConverter.sol

```diff
@@ -70,9 +70,9 @@ contract LiquidityPoolConverter is ConverterBase {
       * can only be called by the contract owner
       * note that prior to version 28, you should use 'acceptTokenOwnership' instead
     */
-    function acceptTokenOwnership() public {

```

Comment:
What about the tests?

Reply:
It's the other way around: this actually fixes the tests 😅